Update README.md with PostgreSQL recommendation

chapb opened the pull request:
Added a recommendation to use PostgreSQL as the database, as the use of MySQL for tt-rss requires InnoDB. This reflects the developers' recommendation, and is simpler to get working for novice users.

<!--- Provide a general summary of your changes in the Title above -->

[linuxserverurl]: https://linuxserver.io
[![linuxserver.io](https://raw.githubusercontent.com/linuxserver/docker-templates/master/linuxserver.io/img/linuxserver_medium.png)][linuxserverurl]
	

<!--- Before submitting a pull request please check the following -->

<!---  That you have made a branch in your fork, we'd rather not merge from your master -->
<!---  That if the PR is addressing an existing issue include, closes #<issue number> , in the body of the PR commit message   -->
<!---  You have included links to any files / patches etc your PR may be using in the body of the PR commit message -->
<!---  -->

##  Thanks, team linuxserver.io

chapb committed:
Update README.md with PostgreSQL recommendation

Added a recommendation to use PostgreSQL as the database, as the use of MySQL for tt-rss requires InnoDB. This reflects the developers' recommendation, and is simpler to get working for novice users.

tobbenb commented:
Could  you please edit the readme-vars.yml instead, as the Readme is built from that file.

chapb committed:
Move README.md fix to parent

Moved the recommendation to use PostgreSQL to the parent file (readme-vars.yml) from which README.md is generated.

chapb commented:
Done, I think!

sparklyballs commented:
innodb has been the standard engine in mariadb since ver 10.2 
https://mariadb.com/kb/en/library/innodb/

and is the default in mysql since ver 5.5

so this is a bit redundant

chapb commented:
Ahh, I was having trouble getting it working with MySQL, and the way the developers emphasized that InnoDB was required when using MySQL in their documentation, I assumed that it was my issue (it worked perfectly after switching from MySQL to PostgreSQL). 

The dev still [recommends it regardless](https://tt-rss.org/wiki/FAQ#what-should-i-use---mysql-or-postgresql) - a better solution may to be removing the reference to InnoDB, but leave the note that PostgreSQL is the preferred choice?
